deploy/sre-prometheus/ocm-agent/obo-monitoring/100-nodes-not-joining-nodepool.PrometheusRule.yaml

```
@@ -16,10 +16,19 @@ spec:
           annotations:
             description: "The NodePool {{ $labels.name }} has not reached its target size for more than 1 hour"
             summary: "NodePool provisioning Failure"
-          expr: sum by (_id, namespace, cluster_name, exported_namespace, name) (((hypershift_nodepools_available_replicas / hypershift_nodepools_size) < 1) unless on (_id) ((hypershift_nodepools_size == 0) or hypershift_cluster_alerts_disabled))
+          expr: |-
+            sum by (_id, cluster_name, exported_namespace, namespace) (
+              (
+                ((hypershift_nodepools_available_replicas / hypershift_nodepools_size) < 1)
+                unless on (_id) (
+                  (hypershift_nodepools_size == 0) or (hypershift_cluster_limited_support_enabled == 1)
+                )
+              )
+              unless on (exported_namespace) last_over_time(hypershift_cluster_deleting_duration_seconds[10m])
+            )
           for: 60m
           labels:
             severity: warning
-            send_managed_notification: "false"
+            send_managed_notification: "true"
```

```suggestion
            send_managed_notification: "false"
```
Please maintain the notification to be false for now.